unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BIWMainController.cs

```diff
@@ -596,7 +569,7 @@ private void EnterEditMode()
         }
 
         CommonScriptableObjects.builderInWorldNotNecessaryUIVisibilityStatus.Set(false);
-        DataStore.i.builderInWorld.showTaskBar.Set(true);
+        DataStore.i.dataStoreBuilderInWorld.showTaskBar.Set(true);
```

Comment:
I see that the `dataStoreBuilderInWorld` change is in here as well. So I think I may be reviewed changes from this PR on #828 . If I find code that already was reviewed there I'll not comment it in this PR, but please check the comments on that one.